### DISCUSSION TITLE
Any plan to support macos, linux and desktop ?

### DISCUSSION THREAD

**Discussion by raghavan-mookiah** [tneotia/html-editor-enhanced#9] [Ideas]
Any plan to support macos, linux and desktop ?

![image](https://user-images.githubusercontent.com/47385593/111926884-9f956500-8ad4-11eb-8ca5-bb8a305cedb9.png)

**Comment by tneotia** [comment]
I hadn't had any plans, but if you think this feature is useful I will consider adding it.

The only contingency is that there must be a way to open a webview in Flutter Desktop. I did some quick googling and I didn't find any indication of how to open a webview on Desktop platforms. I'll look into this in detail a little later but if you have any useful information feel free to post it here!

**Reply by raghavan-mookiah** [reply]
Supporting all platforms will add more value to the package.  If I find any information or idea, I will share that with you.

**Reply by tneotia** [reply]
I did more comprehensive research and I still didn't find anything on Desktop support for webviews.

Looking at your screenshot, Desktop supports `dart:io` and therefore the mobile version of the widget is being loaded. Thus, I also tried to search for webview plugins that support desktop, but none of them do yet. I saw that easy_web_view is [looking into](https://github.com/rodydavis/easy_web_view/issues/17#issuecomment-646710585) support, but its been a while since then.

Without webview/loading an HTML file support in Desktop it just won't be possible to add support, as much as I would like to. Hopefully there is further development on this later down the road.

**Reply by tneotia** [reply]
Quick update: See https://github.com/pichillilorenzo/flutter_inappwebview/issues/460#issuecomment-784213718 for information regarding the roadmap for webview Desktop support.

Edit: Desktop webview support is nearer: https://github.com/pichillilorenzo/flutter_inappwebview/issues/460#issuecomment-820389445

**Reply by emmggi** [reply]
What about other solutions on pub.dev? 

https://pub.dev/packages?q=webview+platform%3Alinux+platform%3Awindows

**Reply by tneotia** [reply]
Unfortunately they cannot be used yet, due to https://github.com/MixinNetwork/flutter-plugins/issues/55

**Reply by emmggi** [reply]
This seems like a really strange situation that there isn't one for desktop. What does it take to make one? Is it a big undertaking?

**Reply by quangdog** [reply]
Has there been any movement on this? We *really* want to use this plugin in our project, but we need Win/osX/iOS support.

**Comment by springeye** [comment]
can use TextSpan for flutter???
